main.tf

```diff
@@ -9,20 +9,22 @@ module "s3_bucket" {
   enabled            = true # Create resources
   versioning_enabled = false
 
-  name               = "firehose-logs"
-  namespace          = "sym"
-  stage              = var.environment
+  name      = "firehose-logs"
+  namespace = "sym"
+  stage     = var.environment
 
   # S3 Bucket names must be globally unique across all AWS accounts. Suffix the account ID to ensure uniqueness
-  bucket_name        = "${lower(var.name_prefix)}sym-firehose-logs-${var.environment}-${data.aws_caller_identity.current.account_id}"
+  bucket_name = "${lower(var.name_prefix)}sym-firehose-logs-${var.environment}-${data.aws_caller_identity.current.account_id}"
 
-  additional_tag_map = var.tags
+  tags = var.tags
```

Comment:
Looking at the docs I don't really understand in what scenario you might want `additional_tag_map` instead of `tags` (though `tags` obv seems like the more commonly useful one). Should we include both?

Reply:
Tbh, I'm uncertain what `additional_tag_map` does. It doesn't actually tag the resource?

Reply:
eh, yeah, ok. the docs say "rare cases" anyway so this is prob fine